access reviews to v1 - using doc stub generator (#12207)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/disableanddeleteuserapplyaction.md

---
title: "disableAndDeleteUserApplyAction resource type"

> description: "Disable and delete is a complex type of accessReviewApplyAction that indicates disabling any B2B guest user who is denied in an access review for 30 days, then subsequently deleting their account."

Reviewer: ```suggestion
description: "Disable and delete a accessReviewApplyAction.."
```

author: "isabelleatmsft"
localization_priority: Normal
ms.prod: "governance"
doc_type: resourcePageType
---

# disableAndDeleteUserApplyAction resource type

Namespace: microsoft.graph

> Disable and delete is a complex type of [accessReviewApplyAction](accessreviewapplyaction.md) that indicates disabling any B2B guest user who is denied in an access review for 30 days, then subsequently deleting their account. This option does not contain any configuration options.

Reviewer: ```suggestion
The [accessReviewApplyAction](accessreviewapplyaction.md) object disables a B2B guest user who is denied in an access review for 30 days, and then deletes their account.
```
